vscode/rootfs/var/lib/code-server/defaults/core/example-mqtt.sh

```
@@ -0,0 +1,62 @@
+    show_vars
+    export -f mqtt_push
+}
+
+git_src() {
+    if [ ! -d "$openmqtt_src_files" ]; then
+        git clone "$openMQTT_git" "$openmqtt_src_files"|| bashio::log.warning "Failed to git clone"
+    else
+        if [ "$PULL" = true ]; then
+            cd "$openmqtt_src_files" || bashio::log.warning "Failed to change dir to $openmqtt_src_files"
+            git pull || bashio::log.warning "Failed to git pull"
+            cd "$CUSTOM_BASH_SCRIPTS_PATH" || "Failed to change dir back to boot scripts root"
```

Shellcheck (0.10.0)

[warning] CUSTOM_BASH_SCRIPTS_PATH is referenced but not assigned.

(SC2154)